### ISSUE TITLE
Proposal: Remove `all` flag for features

### ISSUE THREAD

**Issue by JakeChampion** [polyfillpolyfill/polyfill-service#655]
We strongly recommend not using the kitchen-sink alias `all` as the bundle will be huge, including massive polyfills like Intl with every single language pack, and will also pick up all polyfills we add in future, even those for experimental features where standards are not yet stable.

The kitchen sink polyfill request is rather large to download, it took my machine on a fast internet connection 12.9 minutes to download and a time-to-first-byte of 4.58 seconds.

I propose we remove the `all` alias in favour of users explicitly declaring their dependencies with individual polyfill names or defined sets of polyfills such as `es5` and `blissfuljs`.

**Comment by triblondon** [comment]
I agree with this in principle, and worth saying that we added `all` before we took in all the Intl craziness. It would be hard to argue that this is removable without moving to a new API version, unfortunately. Also there are a fair number of people using it so we also ought to try and address whatever is driving that behaviour.

Assorted ideas:
1. Add `console.log("Requesting all polyfills from polyfill.io is a really bad idea");` to the `all` response.
2. Operate `all` as a blacklist alias rather than a whitelist one, and mark some polyfills as excluded from it.  Then the Intl locales could be excluded, but new polyfills would automatically be included. It should probably be renamed to `almost-all` but would need to continue to support `all` for compat.
3. Reassess which polyfills should be in the default set, to try and address the demand that leads to use of 'all'.  A good way to start this might be to look at what blissfulljs added their alias to, because they had previously advised their users to use `all`.
